src/work_submitter.ts

```diff
@@ -67,8 +67,16 @@ export class WorkSubmitter extends SubmissionQueue<WorkSubmission> {
         user: userLiquidation.user,
         percent_liquidated: userLiquidation.liquidationPercent,
       });
+      const auctionExists =
+        (await sorobanHelper.loadAuction(userLiquidation.user, AuctionType.Liquidation)) !==
+        undefined;
+      if (auctionExists) {
+        return true;
+      }
```

Comment:
can we add a log here for each submission type?

`User Liquidation already exists ${user} ${pecentage}`